gwdetchar/const.py

```diff
@@ -46,7 +46,8 @@
     ('ER9', Segment(1152136817, 1152255617)),
     ('ER10', Segment(1161907217, 1164499217)),
     ('O2', Segment(1164499217, 1187733618)),
-    ('O3', Segment(1187733618, 2000000000)),
+    ('O3', Segment(1187733618, 1269363618)),
+    ('O4', Segment(1368982818, 2000000000)),
```

Comment:
Do we care that GPS time 1368982818 = Wed May 24 17:00:00 UTC 2023, not 15:00 UTC?  Also, should there be one for ER15?  (I suspect not, since there isn't one for ER11-14; just checking.)

Reply:
Good catch, I'll fix the O4 one. This was a copy and paste error. I was unsure about the ER15 segment since there was none of the ER11-14, as you already mentioned. It couldn't hurt to add it. I will do it.